coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -342,10 +344,44 @@ public void removeFromHierarchy(DevelopmentProposal childProposal) throws Propos
             }
             hierarchyProposal.setHierarchyOriginatingChildProposalNumber(lowestProposalNumber);
             dataObjectService.save(childProposal);
+            dataObjectService.save(hierarchyProposal);
```

Comment:
So I know I just merged this but, why are we saving here? Right after this we make all kinds of changes to the hierarchy proposal again? Should this happen after the sync or should we at least pass in the returns hierarchyProposal thats tied to the transaction?